drop/modules/mae-pipeline/MAE/filterSNVs.sh

```diff
@@ -51,5 +51,15 @@ else  # VCF and BAM have same chromosome format
     rm ${tmp}.tbi
 fi
 
+num_out=$(zcat "${output}" | grep -vc '#' )
+if [ "${num_out}" -eq 0 ]
+then
+  echo "ERROR: no entries after filtering for SNVs"
+  echo "VCF ID: ${vcf_id}"
+  echo "VCF file: ${vcf_file}"
+  echo "BAM file: ${bam_file}"
+  exit 1
```

Comment:
Same as above. Please add suggestions for the user to it. 
1. Do we do not have any heterozygous variants in the VCF file?
2. Is the VCF file not correctly formated?
3. Do we have issues with the chromosome style? (Not sure if this is really a problem in this script)
4.  ....

Reply:
number 3. shouldn't be a problem in this script.
- Check if vcf file exists and its index exists (although the pipeline might not even begin if this is not the case)